scripts/check_sync.py

```diff
@@ -12,7 +12,7 @@
 
 sync_data = deque(maxlen=sync_maxlen)
 last_sync = (None,None)
-last_print_clock = time.clock()
+last_print_clock = time.clock() if sys.version_info.major == 2 else time.perf_counter()
```

Comment:
maybe you can trick python2/python3 with an import construct like this:
https://github.com/ipa320/cob_command_tools/pull/291/files#diff-2cf9575e1c952cdb5041d82e408eaafdd6dd9694eee8026acb6e825d743d2fedR29-R32
```
try:
    from time import clock as perf_counter
except ImportError: 
    pass
```

then use `last_print_clock = perf_counter()` here